tests/framework/base/ModelTest.php

```
@@ -32,451 +50,1245 @@ protected function setUp(): void
+        );
+        self::assertSame(
+            [
+                'account_id',
+                'user_id',
+                'email',
+                'name',
+            ],
+            $model->activeAttributes(),
+            "Should include 'email' and 'name' as active with explicit on rule.",
+        );
     }
```

PHPMD (2.15.0)

[warning] The method testSafeScenarios() has 234 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)